PWGCF/FemtoDream/femtoDreamProducerTask.cxx

```diff
@@ -276,6 +296,14 @@ struct femtoDreamProducerTask {
         if (!v0Cuts.isSelectedMinimal(col, v0, postrack, negtrack)) {
           continue;
         }
+
+        if (ConfRejectITSHitandTOFMissing) {
+          o2PhysicsTrackSelection->SetRequireHitsInITSLayers(1, {0, 1, 2, 3});
```

Comment:
In the next iteration move this to `init` and you save a function call per track